### PR TITLE
[libvorbis] Update to latest commit

### PR DESCRIPTION
**Pull request by evpobr**
* Use latest upsteam
* Support config mode (upstream)
* Remove patch 0002 (merged to upstream)
* Remove `ogg.patch` (not needed)

### CODE REVIEW

File: ports/libvorbis/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: libvorbis
-Version: 1.3.6-9eadecc-3
+Version: 1.3.6-4d963fe-4
```
Comment (NancyLi1013) [inline ports/libvorbis/CONTROL:2]: The version should be `1.3.6-4d963fe `or `2020-04-19`.
Please refer to this [doc](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#version) to learn more about version names.

File: ports/libvorbis/portfile.cmake [hunk 2]
```diff
@@ -2,13 +2,11 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/libvorbis/portfile.cmake:2]: `include(vcpkg_common_functions)` is no longer needed.
Could you please remove this line?

File: ports/libvorbis/portfile.cmake [hunk 3]
```diff
@@ -17,6 +15,10 @@ vcpkg_configure_cmake(
 )
 
 vcpkg_install_cmake()
+vcpkg_fixup_cmake_targets(
+    CONFIG_PATH lib/cmake/Vorbis
+    TARGET_PATH share/Vorbis
+)
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
```
Comment (NancyLi1013) [inline ports/libvorbis/portfile.cmake:24]: Could you please update
`configure_file(${SOURCE_PATH}/COPYING ${CURRENT_PACKAGES_DIR}/share/libvorbis/copyright COPYONLY)`
as
`configure_file(${SOURCE_PATH}/COPYING ${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright COPYONLY)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/libvorbis/portfile.cmake (hunk 2 above)
Comment (evpobr) [reply ports/libvorbis/portfile.cmake:2]: Removed.

File: ports/libvorbis/CONTROL (hunk 1 above)
Comment (evpobr) [reply ports/libvorbis/CONTROL:2]: Fixed.

File: ports/libvorbis/portfile.cmake (hunk 3 above)
Comment (evpobr) [reply ports/libvorbis/portfile.cmake:24]: Fixed.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now. Thanks for your PR @evpobr .

**Pull request merged**